gammapy/background/tests/test_models.py

```diff
@@ -43,3 +48,82 @@ def test_y_model(self):
         model = self.model.y_model(-30)
         assert isinstance(model, Gaussian1D)
         assert_allclose(model.parameters, [0, -1, 0.4])
+
+
+class TestCubeBackgroundModel():
+
+    @remote_data
+    def test_read(self):
+        
+        # test shape of bg cube when reading a file
+        filename = '../test_datasets/background/bg_cube_model_test.fits'
+        filename = datasets.get_path(filename, location='remote')
+        bg_cube_file = CubeBackgroundModel.read(filename)
+        assert len(bg_cube_file.background.shape) == 3
+
+
+    @remote_data
+    def test_image_plot(self):
+
+        filename = '../test_datasets/background/bg_cube_model_test.fits'
+        filename = datasets.get_path(filename, location='remote')
+        bg_cube_model = CubeBackgroundModel.read(filename)
+
+        # test bg rate values plotted for image plot of energy bin conaining E = 2 TeV
+        energy = Quantity(2., 'TeV')
+        fig_image, ax_im, image_im = bg_cube_model.plot_images(energy)
+        plot_data = image_im.get_array()
+
+        # get data from bg model object to compare
+        energy_bin, energy_bin_edges = bg_cube_model.find_energy_bin(energy)
+        model_data = bg_cube_model.background[energy_bin]
+
+        # test if both arrays are equal
+        decimal = 4
+        np.testing.assert_almost_equal(plot_data, model_data.value, decimal)
```

Comment:
Please use
```python
from numpy.testing import assert_allclose
```
at the top and then just use `assert_allclose` here.
This makes the lines shorter, i.e. you have more space for the arguments of assert_allclose.

In my experience `assert_allclose` covers all use cases for numerical asserts, so I prefer to use it consistently everywhere instead of using the dozens of other available asserts from pytest or numpy in various places.

Can you please add a very short entry to the Gammapy development guidelines so that I can just point there in the future?
https://gammapy.readthedocs.org/en/latest/development/index.html
I don't think an example is needed, just create an intersphinx link to `numpy.testing.assert_allclose`.